task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -72,6 +79,9 @@ spec:
       requests:
         memory: 512Mi
         cpu: 250m
+    volumeMounts:
+      - name: shared-dir
+        mountPath: /index-build-data
```

Comment:
Nitpick: you can put this in the `stepTemplate` instead to avoid having to copy it to every step